views/heading.html

```diff
@@ -14,7 +14,7 @@
                 <a class="navbar-brand" href="http://localhost:3000/homepage"><span
                         class="glyphicon glyphicon-home"></span>Home</a>
                 <a class="navbar-brand" href="http://localhost:3000/LoginForm"><span
-                        class="glyphicon glyphicon-log-in"></span> Login</a>
+                        class="glyphicon glyphicon-log-in"></span> Logout</a>
```

Comment:
this doesn't look right?

Reply:
that is supposed to be a Logout button because you cant access the nav-bar without logging in first